integration/other_components/cmd_message_test.go

```diff
@@ -62,6 +62,14 @@ func TestGenerateAnAppWithMessage(t *testing.T) {
 		envtest.ExecShouldError(),
 	))
 
+	env.Must(env.Exec("should prevent creating a message whose name only differs in capitalization",
+		step.NewSteps(step.New(
+			step.Exec(envtest.IgniteApp, "s", "message", "--yes", "do-Foo", "bar"),
```

Comment:
@aljo242 I tried to make the change as minimal as possible so I just capitalized the already existing name that the test defined (instead of making a fresh message only to check for the error case).